stories/misc/shortcuts.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Meta } from '@storybook/react';
+import { KbsProvider, useKbsGlobal } from 'react-kbs';
+
+import {
+  RoiContainer,
+  RoiList,
+  RoiProvider,
+  TargetImage,
+  useActions,
+} from '../../src';
+import { Layout } from '../utils/Layout';
+import { initialRois } from '../utils/initialRois';
+
+export default {
+  title: 'Misc',
+  decorators: [
+    (Story) => (
+      <KbsProvider>
+        <RoiProvider initialConfig={{ rois: initialRois }}>
+          <Story />
+        </RoiProvider>
+      </KbsProvider>
+    ),
+  ],
+} as Meta;
+
+export function Shortcuts() {
+  const { cancelAction } = useActions();
+
+  useKbsGlobal([
+    {
+      shortcut: ['Escape'],
+      handler: cancelAction,
+    },
+  ]);
+
+  return (
+    <>
+      <div style={{ height: 1000 }} />
+      <Layout>
+        <RoiContainer target={<TargetImage src="/barbara.jpg" />}>
+          <RoiList />
+        </RoiContainer>
+      </Layout>
+    </>
+  );
```

Comment:
If we come back to this story in one year, we won't be able to know what it's supposed to do without digging into the code.

Please add some text that explains how the story can be tested.